### ISSUE TITLE
Update `GetEmbeddedKernelResources`

### ISSUE THREAD

**Issue by theRoughCode** [microsoft/iqsharp#197]
With [this new PR](https://github.com/microsoft/jupyter-core/pull/64) merged into Jupyter Core, we can remove the "hack" in [here](https://github.com/microsoft/iqsharp/blob/master/src/Tool/Program.cs#L45) to allow proper directory structure and naming with `/`.

**Comment by rmshaffer** [comment]
@theRoughCode releases of [Jupyter Core](https://github.com/microsoft/jupyter-core) are handled manually on an as-needed basis; there's no regular release cadence for that package. Let me know if you'd like more details on that, and I'd be happy to help get a new release pushed.

**Comment by theRoughCode** [comment]
Sounds good! I guess the change doesn't impact any existing files, so we could wait for the next release (whenever that might be). Or if you think that it would be useful to include that change, we can do that and I can work on the fix. :)

**Comment by rmshaffer** [comment]
Makes sense! I think if this doesn't affect anything immediately then there's no need to do it right now. We can just leave this issue open as a reminder to do it after the next release of Jupyter Core.
